main.py

```diff
@@ -1,10 +1,30 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as player_file:
+        player_data = json.load(player_file)
+    for player_name in player_data.keys():
+        info = player_data[player_name]
+        email = info["email"]
+        bio = info["bio"]
+        guild = info["guild"]
```

Comment:
Consider using the `.get()` method to safely access dictionary keys. For example: `guild = info.get('guild')`. This aligns with the checklist recommendation to use `.get()` for dictionary key access.